CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Airbase 140
+
+* Pass `user.timezone` in tests as a system property, not as a JMV argument
```

Comment:
what does JMV stand for?

also mention that this supposedly restores IntelliJ ability to set timezone correctly when running tests from within that IDE.